File: consalvi-tyler/shortest-word.js
/*
String is never empty
Need shortest words
Loop through x
convert to an array (split?) x
test each word for length x
push values into new array
compare lengths (nice)
return samllest value
*/

function findShort(s) {
  let arr = s.split(' ');
  let newArr = [];
  for (let i = 0; i < arr.length; i++) {   
    newArr.push(arr[i].length);
  } return Math.min(...newArr);
}

File: montague-morgan/shortest_word.py
# given string of words, never will be empty
# need so split the words by split
# make the key, with length
# sort the list of words by length using the key
# return length of word at position 0

def length(e):
    return len(e)

def find_short(s):
    x = s.split(' ')
    x.sort(key=length)
    return len(x[0])

    
#     pass l # l: shortest word length
